ports/rp2/modmachine.c

```diff
@@ -228,7 +228,7 @@ static void mp_machine_lightsleep(size_t n_args, const mp_obj_t *args) {
             #if PICO_RP2040
             clocks_hw->sleep_en1 |= CLOCKS_SLEEP_EN1_CLK_USB_USBCTRL_BITS;
             #elif PICO_RP2350
-            clocks_hw->sleep_en1 |= CLOCKS_SLEEP_EN1_CLK_SYS_USBCTRL_BITS;
+            clocks_hw->sleep_en1 |= CLOCKS_SLEEP_EN1_CLK_USB_BITS;
```

Comment:
With this fix, it looks like we can enable the `tests/ports/rp2/rp2_lightsleep.py` test on rp2350, by removing the following lines in that file:
```py
# RP2350 currently fails this test, needs further investigation.
if "RP2350" in sys.implementation._machine:
    print("SKIP")
    raise SystemExit
```

Reply:
I thought we'd removed the lightsleep test due to it not working on pico2? Unfortunately I don't think this has changed. I'll check however.

Reply:
I tested this test using this PR and it works on all Pico boards (Pico, Pico2, Pico2 W, and in RISCV mode).

It tests USB staying alive during lightsleep, not the duration of lightsleep itself.